### PR TITLE
source: add buttons and always show budget

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

**Comment by lepistone** [comment]
Thanks for fixing my bug @gurneyalex. I pushed another UI fix.

**Comment by lepistone** [comment]
this is actually green, but an irrelevant build was cancelled. :green_apple:

**Comment by lepistone** [comment]
I just restarted the build.

File: framework_agreement_sourcing/view/requisition_view.xml [hunk 1]
```diff
@@ -24,14 +33,6 @@
                position="attributes">
           <attribute name="attrs">{'invisible': [('procurement_method', '=', 'fw_agreement')]}</attribute>
         </field>
-        <field name="unit_cost"
```
Comment (nbessi) [inline framework_agreement_sourcing/view/requisition_view.xml:27]: is this onchange related?

**Comment by lepistone** [comment]
@nbessi the last commit made prices always visible, and that's intentional.

I changed the PR title to say that.

**Comment by nbessi** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:
